Command/AbstractCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace winzou\Bundle\StateMachineBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\Question;
+
+abstract class AbstractCommand extends ContainerAwareCommand
```

Comment:
I don't see the point of doing an `AbstractCommand` here, can you merge everything in a single command file? It will be simplier, and simplier is better ;)